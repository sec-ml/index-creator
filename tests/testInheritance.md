| term   | sub-term | notes      | book | page |
| ------ | -------- | ---------- | ---- | ---- |
| term a |          | some notes |      |      |
| term b |          | ^^         |      |      |
| term c | term d<> | ^^         |      |      |
