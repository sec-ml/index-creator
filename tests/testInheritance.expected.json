[
  {
    "term": "term a",
    "subTerm": "",
    "notes": "some notes",
    "book": 0,
    "page": ""
  },
  {
    "term": "term b",
    "subTerm": "",
    "notes": "some notes",
    "book": 0,
    "page": ""
  },
  {
    "term": "term c",
    "subTerm": "term d",
    "notes": "some notes",
    "book": 0,
    "page": ""
  },
  {
    "term": "term d",
    "subTerm": "term c",
    "notes": "some notes",
    "book": 0,
    "page": ""
  }
]
